packages/react-icon-provider/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@fluentui/react-icon-provider",
+  "version": "0.1.0",
+  "description": "Package for applying icon overrides to new svg icons",
+  "private": true,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "test": "just-scripts test",
+    "update-api": "just-scripts update-api"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@fluentui/scripts": "^1.0.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6"
+  },
+  "dependencies": {
+    "@fluentui/set-version": "^8.0.0-beta.0",
+    "@fluentui/style-utilities": "^8.0.0-beta.11",
+    "enzyme": "~3.10.0",
```

Comment:
This needs to be under devDependencies, not dependencies.

Reply:
All three of those packages should be in devDependencies?

Reply:
No just enzyme. How to tell is if the dep is only used for something related to build infrastructure (like enzyme, which helps with tests), it's a devDependency. If it's used at runtime (like set-version and style-utilities), it's a dependency. Special case is React, which is specified for consumers as a peerDependency (to give more flexibility on which version consumers can use), so that also goes as a devDependency locally.